packages/eslint-plugin-query/package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "@tanstack/eslint-plugin-query",
+  "version": "1.0.0",
+  "description": "ESLint plugin for React Query",
+  "author": "Eliya Cohen",
+  "license": "MIT",
+  "repository": "tanstack/query",
+  "homepage": "https://tanstack.com/query",
+  "funding": {
+    "type": "github",
+    "url": "https://github.com/sponsors/tannerlinsley"
+  },
+  "main": "build/lib/index.js",
+  "scripts": {
+    "typecheck": "tsc",
+    "build": "tsup --minify",
+    "dev": "tsup --watch --sourcemap",
+    "test:jest": "../../node_modules/.bin/jest --config ./jest.config.ts",
+    "test:dev": "pnpm run test:jest --watch",
+    "lint:eslint": "eslint --ext .js,.jsx,.ts,.tsx src"
```

it has to be named `test:eslint` like in all the other packages so that `pnpm run test:estlint` picks it up for this package as well:

```suggestion
    "test:eslint": "eslint --ext .js,.jsx,.ts,.tsx src"
```